#issue-1108 Describe Image Detail for products (#1110)
magento/merchdocs: src/catalog/product-images-and-video.md

    - Click the **camera** (![]({% link images/images/btn-camera.png %})) tile, select the image file, and click <span class="btn">Open</span>.

A [placeholder]({% link catalog/product-image-placeholders.md %}) image appears in the catalog until a product image is available.

## Image Detail

> When product images are added, there is the possibility to customize their displaying in the **Image Detail** window.

Reviewer: ```suggestion
When product images are added, there is the possibility to customize the display in the _Image Detail_ window.
```

 ![]({% link images/images/image-video.png %}){: .zoom}
    _Product images_

> Set image displaying options for a product:

Reviewer: ```suggestion
To set image displaying options for a product:
```

> 1. Click on the product image.

Reviewer: ```suggestion
1. Click the product image.
```

1. Enter the **Alt text**.

1. Choose a **Role** for displaying the image (all roles can be selected for one image):

    - **Swatch**
    - **Base**
    - **Small**
    - **Thumbnail**

> All of these roles are selected for the first added image by default.

Reviewer: ```suggestion
   All of these roles are selected for the first added image by default.
```

>  ![]({% link images/images/details.png %}){: .zoom}

Reviewer: ```suggestion
   ![]({% link images/images/details.png %}){: .zoom}
```

    _Image details_

> To hide the product image from the product page select the **Hide from Product Page** checkbox.

Reviewer: ```suggestion
To hide the product image from the product page, select the **Hide from Product Page** checkbox.
```
